tests/unit/models/gpflow/test_models.py

```diff
@@ -568,6 +570,83 @@ def test_gaussian_process_regression_optimize(compile: bool) -> None:
     assert model.model.training_loss() < loss
 
 
+@random_seed
```

Comment:
should we also test that the posterior objects update properly? or maybe since this is the new default it is already tested elsewhere?